README.md

```diff
@@ -134,6 +136,13 @@ Learn more about ONNX
 - [ONNX website](http://onnx.ai/)
 - [ONNX on GitHub](https://github.com/onnx/onnx)
 
+### Platforms supported
+| OS/Browser | Chrome | Edge | FireFox | Safari | Opera |
```

Comment:
We also have a line in ./docs/operators.md about current platform support being Windows 10 +Chrome/... Can we remove that line and just add a link to ./compatibility.md there too ?

Reply:
yes. since we already have 2 files this is good idea